asv/benchmarks/simulation/bench_viewer.py

```
@@ -0,0 +1,127 @@
+        builder = Example.create_model_builder(robot, num_worlds, randomize=True, seed=123)
+
+        # finalize model
+        self._model = builder.finalize()
+        self._state = self._model.state()
+
+        # Setting up the renderer
+        self.renderer = ViewerGL(headless=True)
+        self.renderer.set_model(self._model)
+
+    @skip_benchmark_if(wp.get_cuda_device_count() == 0)
+    def time_rendering_frame(self, robot, num_worlds):
```

Ruff (0.14.1)

Unused method argument: `robot`

(ARG002)